### PR TITLE
RHPAM-3032: getting UI error if navtree.json file is not available in…

### PR DESCRIPTION
**Pull request by jesuino**
… the imported dashboard

### CODE REVIEW

**Comment by sthundat** [comment]
Jenkins execute full downstream build

File: dashbuilder/dashbuilder-runtime/src/main/java/org/dashbuilder/backend/services/impl/ImportValidationServiceImpl.java [hunk 1]
```diff
@@ -35,32 +35,36 @@
...
-
+        
         return false;
     }
 
     private boolean entryStartsWith(ZipEntry entry, String path) {
         return entry.getName().startsWith(path);
     }
 
+
```
Comment (sthundat) [inline dashbuilder/dashbuilder-runtime/src/main/java/org/dashbuilder/backend/services/impl/ImportValidationServiceImpl.java:69]: Please remove the empty line

Comment (jesuino) [reply dashbuilder/dashbuilder-runtime/src/main/java/org/dashbuilder/backend/services/impl/ImportValidationServiceImpl.java]: done, thanks Savitha

**Comment by sthundat** [comment]
@jesuino ,I have verified the fix and its working fine

**Pull request merged**